### DISCUSSION TITLE
r/the-important-consideration-when-buying-an-rv

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#67] [Announcements]
# r/the-important-consideration-when-buying-an-rv

三年房车用户教你选房车

https://blog.ops-coffee.cn/r/the-important-consideration-when-buying-an-rv

**Comment by KeyleXiao** [comment]
我前段时间还在跟爱人商量要不出去玩租个房车，又方便，可以在景点附近住下，还可以出门就旅游，在青山绿水环绕间入眠。
哈哈，我觉得你说的很对，低调是最重要的。低调的车身会少很多麻烦。

**Reply by ops-coffee** [reply]
是的，低调点好

**Comment by Icedroid** [comment]
好奇这个房车要多少米

**Reply by ops-coffee** [reply]
https://blog.ops-coffee.cn/r/index.html 这里有详细介绍，落地接近25w，裸车22w
